cmake/FindPythia8.cmake

```diff
@@ -0,0 +1,48 @@
+# Find the Pythia8 includes and library.
```

Comment:
Is this file used? Why do we need a dependency to pythia? 
at least it should be optional.